cmd/broker/testdata/old-sap-converged-cloud-region-mappings.yaml

```diff
@@ -1,2 +1,3 @@
 cf-eu20-staging:
   - eu-de-1
+  - eu-de-2
```

Comment:
this file contains a mapping between platform regions and available openstack regions. It is taken into account in the catalog endpoint which is used by the Pivotal framework. So if the unit test tries to create sapconverged-cloud runtime, but the Platform region does not match the cf-eu20-staging - it fails